source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -16,19 +16,9 @@ Drush-savvy developers should also install and utilize [Terminus](https://github
 Using Terminus to operate Drush commands on your site environments negates the issues below, which stem from incompatibilities between locally and remotely installed versions of Drush. All of the commands below can be run from Terminus instead of using Drush aliases. For more information, see our guide on [Managing Drupal Sites with Terminus and Drush](/docs/guides/terminus-drupal-site-management/).
 
 ## Drush Versions
-Pantheon currently has Drush version 5.10.0 installed; Drush 5.x is compatible. Currently, Pantheon aliases are not Drush 6.x compatible, but we're working on it.
+Pantheon currently has Drush version 5.10.1 installed. You can run Drush 5.x, 7.x and 8.x on your local installation to interact with your Pantheon Drupal installations. Drush 6.x is not supported.
 
-You can run either Drush 5.x or 6.x on your local installation to interact with your Pantheon Drupal installations.
-
-###Known Issues
-
-#### Drush 5
-
-Some Drush 5 commands need to be executed from outside the context of a local working Drupal installation.
-
-#### Drush 6
-
-If your local Drush installation is version 6, you'll need to execute most commands with the `--strict=0` option in order to parse Pantheon alias files.
+See [Greg Anderson's](https://pantheon.io/team/greg-anderson) blog post [Introducing Drush 8](https://pantheon.io/blog/introducing-drush-8) for information on upgrading.

```

Comment:
A better blog to reference would be https://pantheon.io/blog/fix-drush-site-aliases-policy-file, but Timani nails it in his comment.